diff --git a/Makefile.in b/Makefile.in
index 84e52e9..d31156b 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -226,7 +226,7 @@ endif
 CCMD  = $(CC) -c $(COPTS) -I. $(LUA:%=-I%/include) \
               $(CFLAGS) $(QDP_CF) $(AFF_CF) $(HDF5_CF) $(QUDA_CF) $(CLOVER_CF) $(TWISTED_CF) $(MDWF_CF)\
               $(GSL_CF) $(HYPRE_CF) $(ARPACK_CFLAGS) $(QOPQDP_CF)
-LCMD  = $(LD) $(COPTS) $(LDFLAGS) $(LUA:%=-L%/lib) \
+LCMD  = $(LD) $(LDFLAGS) $(LUA:%=-L%/lib) \
               $(QDP_LF) $(AFF_LF) $(HDF5_LF) $(QUDA_LF) $(CLOVER_LF) $(TWISTED_LF) $(MDWF_LF) $(HYPRE_LF) \
 	      $(ARPACK_LDFLAGS) $(QOPQDP_LF)
 LLIBS = $(QOPQDP_LIBS) $(QUDA_LIBS) \
@@ -263,6 +263,7 @@ install: all doc
 ###############################################################################
 $(program): $(drvobj) $(library)
 	$ELD $(program)
+	$E$(LCMD) -o $(program) $(drvobj) -L. $(library:lib%.a=-l%) $(LLIBS)
 	$C$(LCMD) -o $(program) $(drvobj) -L. $(library:lib%.a=-l%) $(LLIBS)
 
 $(drvobj) $(libobj): %.o: sources/%.c
